### PR TITLE
Add limit parameter to client.data_object.get() REST query

### PR DESCRIPTION
**Pull request by dirkkul**
closes https://github.com/semi-technologies/weaviate-python-client/issues/179

### CODE REVIEW

File: weaviate/data/crud_data.py [hunk 1]
```diff
@@ -433,11 +435,14 @@ def get(
...
             by default False
-        class_name : Optional[str], optional
+        class_name: Optional[str], optional
             The class name of the object with UUID `uuid`. Introduced in Weaviate version v1.14.0.
             STRONGLY recommended to set it with Weaviate >= 1.14.0. It will be required in future
             versions of Weaviate Server and Clients. Use None value ONLY for Weaviate < v1.14.0,
             by default None
+        limit: Optional[int], optional
+            The maximum number of data objects to return.
+            by default None
```
Comment (StefanBogdan) [inline weaviate/data/crud_data.py:445]: maybe we should mention that `None` would result in Weaviate Server default value which is 100.

File: weaviate/data/crud_data.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 from requests.exceptions import ConnectionError as RequestsConnectionError
 
+from weaviate.batch.crud_batch import _check_positive_num
```
Comment (StefanBogdan) [inline weaviate/data/crud_data.py:10]: Should we move this function to `weaviate.util`?

**Review by StefanBogdan — COMMENTED** [review]
BTW there are more parameters: `offset`, `sort`, `order` (can be `"asc"` or `"desc"). These are the same parameters as for `Get` query using GraphQL. Here is how it is done there:
- Sort/order: https://github.com/semi-technologies/weaviate-python-client/blob/main/weaviate/gql/get.py#L797-L859
- Offset: https://github.com/semi-technologies/weaviate-python-client/blob/main/weaviate/gql/get.py#L486-L512

I am not sure anyone uses them, to be honest, because with `GET /objects` you cannot do any filtering, so I am not sure anyone is interested in this. Also with `sort`/'order' I am not sure how it would work when you have multiple classes with different properties. This was kind of the reason I never added `limit`, and these ones I mentioned here.

Do you think we should include those as well?

File: weaviate/data/crud_data.py (hunk 1 above)
Comment (dirkkul) [reply weaviate/data/crud_data.py]: good point, added :)

File: weaviate/data/crud_data.py (hunk 2 above)
Comment (dirkkul) [reply weaviate/data/crud_data.py]: yes, done

**Comment by dirkkul** [comment]
> Do you think we should include those as well?

Hmmm, if you don't think that anybody uses it, lets wait until people ask for it.

**Review by StefanBogdan — APPROVED** [review]

**Comment by samos123** [comment]
as a user, I was expecting all parameters to be available and thought I must be missing something. I could use the order parameter as well. Right now I sort the results myself in python for one of the test cases in a connector :facepalm:

**Comment by dirkkul** [comment]
ok, I'll create an issue for this. But I will probably not have for it this year

**Pull request merged**